server/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "dev": "DOTENV_CONFIG_PATH=../config/dev.env nodemon --ignore './scripts/' -r dotenv/config server.js --watch",
     "dev-debug": "DEBUG=express:* DOTENV_CONFIG_PATH=../config/dev.env nodemon --inspect -r dotenv/config server.js --watch",
-    "jest": "SEQUELIZE_TEST_HELPERS_JEST_RUNNER=true dotenv -e ../config/test.env -- jest",
+    "jest": "SEQUELIZE_TEST_HELPERS_JEST_RUNNER=true dotenv -e ../config/test.env -- jest --detectOpenHandles --forceExit",
```

Comment:
Nope, this is intentional for the moment due to a performance hang inside `server/tests/integration/authentication.test.js` that happens since the move to using the GitHub Action CI in place of Travis.
We can probably create a separate issue for this!

Reply:
Oh no, that test is causing issues? I'll fix it when I get a chance.

Reply:
I fixed this! And we merged it into the PR 👍